plugins/obs-ffmpeg/obs-ffmpeg-av1.c

```diff
@@ -408,7 +408,8 @@ static bool av1_encode(void *data, struct encoder_frame *frame,
 		debug("cur: %lld, packet: %lld, diff: %lld", cur_ts,
 		      recv_ts_nsec, cur_ts - recv_ts_nsec);
 #endif
-		if (llabs(cur_ts - recv_ts_nsec) > TIMEOUT_MAX_NSEC) {
+
+		if (llabs(cur_ts - (int64_t)recv_ts_nsec) > TIMEOUT_MAX_NSEC) {
```

Comment:
This is the change that I have no idea if it's an good idea to do that or just let the warning.

Reply:
Will be removed on the next push.